OLCNE/scripts/provision.sh

```diff
@@ -391,6 +392,15 @@ certificates() {
   echo_do sed -i -e "'s/^USER=.*/USER=vagrant/'"  ${CERT_DIR}/olcne-tranfer-certs.sh
 
   echo_do bash -e ${CERT_DIR}/olcne-tranfer-certs.sh
+
+  echo_do /etc/olcne/gen-certs-helper.sh --one-cert --nodes "externalip-validation-webhook-service.externalip-validation-system.svc,externalip-validation-webhook-service.externalip-validation-system.svc.cluster.local" --cert-dir "${EXTERNALIP_VALIDATION_CERT_DIR}"
+
+  echo_do sed -i -e "'s/externalip-validation-webhook-service.externalip-validation-system.svc externalip-validation-webhook-service.externalip-validation-system.svc.cluster.local/${nodes//,/ }/'" ${EXTERNALIP_VALIDATION_CERT_DIR}/olcne-tranfer-certs.sh
+
+  echo_do sed -i -e "'s/^USER=.*/USER=vagrant/'" ${EXTERNALIP_VALIDATION_CERT_DIR}/olcne-tranfer-certs.sh
+
+  echo_do bash -e ${EXTERNALIP_VALIDATION_CERT_DIR}/olcne-tranfer-certs.sh
+
```

Comment:
My understanding is that the certificate for the externalIPs kubernetes service should only be on the operator node, there is no need for a transfer.
@mark-au to confirm